### PR TITLE
fix: broken links when tab not specified closes #455

### LINKED ISSUE #455: [Links]: Internal website links not working
## Detailed description

Links for internal pages aren't working

## Steps to reproduce the issue

https://www.carbondesignsystem.com/components/search/usage/#variations
Click link to data table

image

Also not working on other pages as well.
- https://www.carbondesignsystem.com/components/data-table/usage/#search
- https://www.carbondesignsystem.com/components/button/usage/#general-guidance

### PR DESCRIPTION
**Pull request by vpicone**

### CODE REVIEW

File: src/pages/components/link/usage.mdx [hunk 1]
```diff
@@ -6,7 +6,7 @@ tabs: ['Code', 'Usage', 'Style']
 
 ## General guidance
 
-_Links_ are used primarily as a navigational element. Links may also change what or how data is displayed (i.e., view more, show all). If the action taken by the user will change or manipulate data, use a [button](/components/button).
+_Links_ are used primarily as a navigational element. Links may also change what or how data is displayed (i.e., view more, show all). If the action taken by the user will change or manipulate data, use a [button](/components/button/code.
```
Comment (jnm2377) [inline src/pages/components/link/usage.mdx]: same as before

File: src/pages/components/notification/usage.mdx [hunk 2]
```diff
@@ -16,7 +16,7 @@ All notifications have subject titles, which should be short and descriptive. (E
 
 #### Message
 
-We recommend the body of the notification be contained within two lines. Be descriptive and include any troubleshooting actions or next steps. When possible, communicate the main message using just the title. You can include [links](/components/link) within the notification body that redirect the user to next steps.
+We recommend the body of the notification be contained within two lines. Be descriptive and include any troubleshooting actions or next steps. When possible, communicate the main message using just the title. You can include [links](/components/link/code within the notification body that redirect the user to next steps.
```
Comment (jnm2377) [inline src/pages/components/notification/usage.mdx]: same as before

File: src/pages/components/notification/usage.mdx [hunk 3]
```diff
@@ -57,7 +57,7 @@ Toast notifications slide in and out a page from the top-right corner. Actionabl
 
 #### Inline notifications
 
-Inline notifications appear near its related item. In [forms](/components/form), we recommend placing the inline notification at the bottom of the form, right before the submission buttons. Depending on the context of the page, inline notifications can appear above the content as well.
+Inline notifications appear near its related item. In [forms](/components/form/code, we recommend placing the inline notification at the bottom of the form, right before the submission buttons. Depending on the context of the page, inline notifications can appear above the content as well.
```
Comment (jnm2377) [inline src/pages/components/notification/usage.mdx]: same as before

File: src/pages/components/pagination/usage.mdx [hunk 4]
```diff
@@ -18,7 +18,7 @@ Clearly identify which page the user is on by displaying the current page number
 
 #### Provide various options for navigating
 
-_Previous_ and _next_ chevrons or links are the most useful way for the user to move forward or backward through pages of data. Provide an [inline select](/components/select) in which users can choose the page they wish to navigate to.
+_Previous_ and _next_ chevrons or links are the most useful way for the user to move forward or backward through pages of data. Provide an [inline select](/components/select/code in which users can choose the page they wish to navigate to.
```
Comment (jnm2377) [inline src/pages/components/pagination/usage.mdx]: same as before

File: src/pages/components/select/usage.mdx [hunk 5]
```diff
@@ -57,7 +57,7 @@ Labels are essential to the usability of forms. Do not place a label inside a _s
 
 ### Validation
 
-Real-time validation helps to streamline the process and keep data clean when the user is filling out forms. For full guidelines, refer to the [forms](/components/form) usage page.
+Real-time validation helps to streamline the process and keep data clean when the user is filling out forms. For full guidelines, refer to the [forms](/components/form/code usage page.
```
Comment (jnm2377) [inline src/pages/components/select/usage.mdx]: same as before

File: src/pages/components/tile/usage.mdx [hunk 6]
```diff
@@ -10,15 +10,15 @@ Tiles are a highly flexible component for displaying a wide variety of content,
 
 Carbon ships a basic tile structure that responds to the grid. Tiles have no pre-set styles for the content within them. You can customize tiles to fit your specific use case.
 
-When using a call-to-action (CTA) within a tile, use a [secondary button](/components/button). Primary buttons should be reserved for the most important action a user can take on the page.
+When using a call-to-action (CTA) within a tile, use a [secondary button](/components/button/code. Primary buttons should be reserved for the most important action a user can take on the page.
```
Comment (jnm2377) [inline src/pages/components/tile/usage.mdx]: same as before

File: src/pages/components/tile/usage.mdx [hunk 7]
```diff
@@ -10,15 +10,15 @@ Tiles are a highly flexible component for displaying a wide variety of content,
...
 
 ## Variations
 
 _All of the images below represent examples of types of content that could be presented within a tile. The styling in these examples is for illustration only; you are free to create your own layout and design within a tile._
 
 ### Read-only
 
-Read-only tiles are used to display information to the user, such as features or services offered. Read-only tiles are often seen on marketing pages to promote content. These tiles can have internal calls-to-action (CTAs), such as a [button](/components/button) or a [link](/components/link).
+Read-only tiles are used to display information to the user, such as features or services offered. Read-only tiles are often seen on marketing pages to promote content. These tiles can have internal calls-to-action (CTAs), such as a [button](/components/button/code or a [link](/components/link/code.
```
Comment (jnm2377) [inline src/pages/components/tile/usage.mdx]: same for both of these links

File: src/pages/components/tile/usage.mdx [hunk 8]
```diff
@@ -42,7 +42,7 @@ Clickable tiles can be used as navigational items, where the entire tile is a cl
 
 ### Selectable
 
-Selectable tiles work like a [radio button](/components/radio-button), where the entire tile is a click target. Selectable tiles may contain internal CTAs (like links to docs) if the internal CTA is given its own click target. Selectable tiles work well for presenting options to a user in a structured manner, such as a set of pricing plans.
+Selectable tiles work like a [radio button](/components/radio-button/code), where the entire tile is a click target. Selectable tiles may contain internal CTAs (like links to docs/code if the internal CTA is given its own click target. Selectable tiles work well for presenting options to a user in a structured manner, such as a set of pricing plans.
```
Comment (jnm2377) [inline src/pages/components/tile/usage.mdx]: _(like links to docs/code_ is missing the closing `)`

File: src/pages/get-started/design/index.mdx [hunk 9]
```diff
@@ -150,7 +150,7 @@ To toggle the grid on and off, use `Control+G`. To toggle the layout guide, use
 
 ### Symbols
 
-Carbon [components](/components/overview), [add-ons](/contributing/add-ons), and [icons](/guidelines/icons/library) live in the design kit as Sketch _symbols_.
+Carbon [components](/components/overview/code, [add-ons](/contributing/add-ons), and [icons](/guidelines/icons/library) live in the design kit as Sketch _symbols_.
```
Comment (jnm2377) [inline src/pages/get-started/design/index.mdx]: same as before

**Comment by alisonjoseph** [comment]
Oops. I merged too soon 🤦‍♀  @vpicone can you fix the links above and we can merge in asap

File: src/pages/patterns/overflow-content/index.mdx [hunk 10]
```diff
@@ -153,7 +153,7 @@ Mid-line truncation does not have its own class as it requires JavaScript. This
 
 ### 'Show more' buttons
 
-The ‘Show more’ button is used when there is a significant amount of overflow content. Implementing a ‘Show more’ button gives a user the ability to see the content in more digestible chunks, as opposed to all at once. A ‘Show more’ button is used in place of scrolling, gradients, or fades as they are more prominent and actionable. If needed, a 'Show less' can be used to again hide the content the user opened. ‘Show more’ can also be presented as ‘Load more’ in cases where performance is a concern. See the [Loading](/components/loading) section for additional details.
+The ‘Show more’ button is used when there is a significant amount of overflow content. Implementing a ‘Show more’ button gives a user the ability to see the content in more digestible chunks, as opposed to all at once. A ‘Show more’ button is used in place of scrolling, gradients, or fades as they are more prominent and actionable. If needed, a 'Show less' can be used to again hide the content the user opened. ‘Show more’ can also be presented as ‘Load more’ in cases where performance is a concern. See the [Loading](/components/loading/code section for additional details.
```
Comment (jnm2377) [inline src/pages/patterns/overflow-content/index.mdx]: this link is also missing `)`

File: src/pages/updates/whats-new/index.mdx [hunk 11]
```diff
@@ -363,7 +363,7 @@ Sign up for a [Carbon Sessions time slot](https://github.com/carbon-design-syste
 
 #### Dropdown
 
-We're excited to bring some added functionality that enhances our [Dropdown](/components/dropdown) component. Filtering, Multi-select, and Inline Dropdown are new additions to our basic Dropdown component.
+We're excited to bring some added functionality that enhances our [Dropdown](/components/dropdown/code component. Filtering, Multi-select, and Inline Dropdown are new additions to our basic Dropdown component.
```
Comment (jnm2377) [inline src/pages/updates/whats-new/index.mdx]: same as before

File: src/pages/updates/whats-new/index.mdx [hunk 12]
```diff
@@ -379,7 +379,7 @@ _Note: This component is currently only available in [React](http://react.carbon
 
 #### Small Toggle
 
-Small Toggle is a style variation of our traditional [Toggle](/components/toggle) component. It's more compact in size, which allows them to be used in Data Tables or inline with Form elements.
+Small Toggle is a style variation of our traditional [Toggle](/components/toggle/code component. It's more compact in size, which allows them to be used in Data Tables or inline with Form elements.
```
Comment (jnm2377) [inline src/pages/updates/whats-new/index.mdx]: same as before

**Comment by alisonjoseph** [comment]
I cancelled the deployment of this PR so we can fix links

**Pull request merged**